app/views/withdraw/new.html.erb

```diff
@@ -19,10 +19,11 @@
   <% end %>
 
   <%= render "govuk_publishing_components/components/contextual_guidance", {
-    id: "withdrawal-public-explanation-guidance",
+    id: "withdrawal-public-explanation",
```

Comment:
Why has the id changed?

Reply:
I thought would be good to align this with all the other ones so they consistently have the `guidance_id` with a `-guidance` suffix and the component to use a more generic name.